cmd/zoekt-dynamic-indexserver/main.go

```diff
@@ -132,7 +137,23 @@ func (s *indexServer) serveHealthCheck(w http.ResponseWriter, r *http.Request) {
 	// Nothing to do. Just return 200
 }
 
+var (
+	promRegistry        = prometheus.NewRegistry()
+	metricRequestsTotal = promauto.With(promRegistry).NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "zoekt_dynamic_indexserver_requests_total",
+			Help: "Total number of HTTP requests by status code and method.",
+		},
+		[]string{"code", "method"},
```

Comment:
I wonder if we should add `path` to this as well? Since we only have 1 endpoint now it's not that important but it's at least as important as `method` I think.